classrom.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './student'
+
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  has_many :student
```

Comment:
- [ ] I see you're using active record associations in a plain ruby app. Please note that this will throw an error especially since there's not a Gemfile that includes the ActiveRecord gem.
If you want to use ActiveRecord in a regular Ruby app. Please follow this [guide](https://www.techcareerbooster.com/blog/use-activerecord-in-your-ruby-project#:~:text=ActiveRecord%20is%20a%20gem%20that,RDBMS%2C%20like%20MySQL%20or%20PostgreSQL.).